base_comment_template/tests/fake_models.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2017 LasLabs Inc.
+# Copyright 2018 ACSONE
+# Copyright 2018 Camptocamp
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from odoo import models
+
+
+def setup_test_model(env, model_cls):
+    """Pass a test model class and initialize it.
+
+    Courtesy of SBidoul from https://github.com/OCA/mis-builder :)
```

Comment:
this dockstring should be kept?

Reply:
It's exists now in 13.0, IMO kep according commits.